api/serverInfo.php

```diff
@@ -0,0 +1,46 @@
+<?php
+header('Content-Type: application/json');
+
+ob_start();
+require_once 'config.php';
+$output = ob_end_clean();
+
+$content = $_GET['content'];
+
+switch ($content) {
+    case 'nav-remotebuzzerlog':
+        echo dumpfile($config['foldersAbs']['tmp'] . '/' . $config['remotebuzzer']['logfile']);
+        break;
+
+    case 'nav-synctodrivelog':
+        echo dumpfile($config['foldersAbs']['tmp'] . '/' . $config['synctodrive']['logfile']);
+        break;
+
+    case 'nav-myconfig':
+        print_r($config);
+        break;
+
+    case 'nav-serverprocesses':
+        echo shell_exec('/bin/ps -ef');
+        break;
+
+    case 'nav-bootconfig':
+        echo dumpfile('/boot/config.txt');
```

Comment:
If Photobooth is installed on default path, this will look at
`/var/www/htm/boot/config.txt` or not?
Guess we need to use the exec function of php to get the content of `/boot/config.txt`

Reply:
It works as intended if installed in a subdirectory to/var/www/html

Did you try it out ? I understand your thinking but from my testing it actually works.

Reply:
Does work, you're right. was a wild guess.